### DISCUSSION TITLE
Module not found: Can't resolve "<internal_package>"

### DISCUSSION THREAD

**Discussion by lpbonomi** [vercel/turborepo#7888] [Help]
### Summary

I'm trying to import an internal package to my main NextJS app but I can't.

I've followed the docs from turborepo but I can't fix it.

### Additional information

```js
internal package package.json:


{
  "name": "database-helper",
  "version": "1.0.0",
  "prisma": {
    "seed": "tsx seed.ts"
  },
  "scripts": {
    "checkCompile": "tsc --noEmit --skipLibCheck"
  },
  "type": "module",
  "dependencies": {
    "@prisma/client": "^5.8.1",
    "aws-helper": "*"
  },
  "devDependencies": {
    "prisma": "^5.8.1",
    "tsx": "^3.12.10"
  },
  "exports": {
    ".": "./index.js"
  }
}
```

internal package tsconfig.json:

```
{
  "compilerOptions": {
    "target": "ES2022",
    "module": "NodeNext",
    "allowJs": true,
    "outDir": "dist",
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true,
    "strict": true,
    "skipLibCheck": true,
    "declaration": true,
    "composite": true,
    "sourceMap": true,
    "declarationMap": true,
    "verbatimModuleSyntax": true,
    "resolveJsonModule": true,
    "moduleDetection": "force",
    "noUncheckedIndexedAccess": true,
    "lib": ["ES2022"],
    "noUnusedParameters": true,
    "noUnusedLocals": true,
  },
  "include": ["."],
  "exclude": ["node_modules", "dist", ]
}
```

NextJS config:

```
  transpilePackages: [
    'aws-helper',
    'blockchain-helper',
    'database-helper',
    'eslint-config-custom',
  ],
```

NextJS package.json deps

`"blockchain-helper": "*"`
```


### Example

(https://github.com/lpbonomi/turbo-repro)

**Comment by anthonyshew** [comment]
Please provide a reproduction. This error message can have several different root causes and its easy for the community to help you out with a repro.

**Reply by lpbonomi** [reply]
Hi @anthonyshew , thanks for your reply!

Minimum Reproduction here: https://github.com/lpbonomi/turbo-repro

hope it helps

**Reply by Davane** [reply]
@lpbonomi Have you been able to figure out how to do this?

**Reply by lpbonomi** [reply]
Hey @Davane, I'm sorry but I can't remember what I did to fix it and I don't longer have access to the repo I did the work. My recommendation would be that you started with the the smallest internal package you can and start from there. If it doesn't work, maybe start with a new NextJS project and try it there and compare what's different

**Reply by Davane** [reply]
@lpbonomi Ok thank you
